src/controllers/usersController.js

```diff
@@ -0,0 +1,95 @@
+const userServices = require('../services/usersServices.js');
+
+const listAllUsers = async (req, res) => {
+  try {
+    const users = await userServices.getAll();
+
+    res.status(200).send(users.map((user) => userServices.normalize(user)));
+  } catch (error) {
+    res.status(500).send('Internal Server Error');
+  }
+};
+
+const getUser = async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const user = await userServices.getUserById(id);
+
+    if (!user) {
+      return res.sendStatus(404);
+    }
+    res.status(200).send(user);
+  } catch (error) {
+    res.sendStatus(500);
+  }
+};
+
+const createUser = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.status(400).send('Name is required!');
+  }
+
+  try {
+    const newUser = await userServices.createUser(name);
+
+    res.status(201).json(newUser);
+  } catch (error) {
+    res.status(500).send('Internal Server Error');
+  }
+};
+
+const deleteUser = async (req, res) => {
+  const { id } = req.params;
```

Comment:
Probably it is a good idea to check if the id was sent.
By blocking the request if the user did not send all that was needed, you can prevent the db interaction before the fail and save some processing power and money